engine-tests/src/tests/silo.rs

```diff
@@ -20,10 +21,13 @@ const INITIAL_BALANCE: Wei = Wei::new_u64(10u64.pow(18) * 10);
 const ZERO_BALANCE: Wei = Wei::zero();
 const INITIAL_NONCE: u64 = 0;
 const TRANSFER_AMOUNT: Wei = Wei::new_u64(10u64.pow(18) * 4);
-const FEE: Wei = Wei::new_u64(10u64.pow(18));
+const FIXED_GAS: EthGas = EthGas::new(10u64.pow(18));
+const ONE_GAS_PRICE: Wei = Wei::new_u64(1);
```

Comment:
If we were being very strict about units, a gas price would have units of `Wei / EthGas`, not simply `Wei`. But it is common to talk about Ethereum gas prices in units of `Wei` (ignoring the "per gas" part), so maybe this is ok.